src/components/details/styles.jsx

```diff
@@ -0,0 +1,41 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export const useStylesDesktop = makeStyles((_) => ({
+    mainContainer: {
+        minWidth: '60vw',
+        maxWidth: '60vw',
+        minHeight: '20vw',
+        display: 'flex',
+        flexDirection: 'rows',
+        alignItems: 'center',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/FeNixCrew/MiEntradita-frontend/pullRequest?prid=8421134)
```suggestion
        alignItems: "center",
```